utility.lua

```diff
@@ -129,25 +129,39 @@ hopper.take_item_from = function(hopper_pos, target_pos, target_node, target_inv
 	end
 end
 
-local function send_item_to_air(hopper_inv, target_pos, filtered_items)
+local function send_item_to_air(hopper_inv, target_pos)
+	local item
 	local stack
 	local stack_num
 	for i = 1, hopper_inv:get_size("main") do
 		stack = hopper_inv:get_stack("main", i)
-		local item = stack:get_name()
-		if item ~= "" and (filtered_items == nil or filtered_items[item]) then
+		item = stack:get_name()
+		if item ~= "" then
```

Comment:
Duplicated variable. No need to track `item` separately when you can do it like this:
```Lua
if  not stack:is_empty()
```
The logging function below would then use `stack:get_name()`.

Reply:
Removed.